packages/apps/web-viewer-test/src/components/home/ViewerHome.tsx

```diff
@@ -152,6 +156,7 @@ const ViewerHome: React.FC = () => {
           }),
         ]}
         backstageItems={backstageItems}
```

Comment:
do we need this and provider? i guess prefer the provider over passing directly into the viewer

Reply:
Yes, it needs both: if the backstageItems property is removed, the button can't be clicked. If the provider is removed, the button can be clicked but the panel will be empty.

Reply:
ok this seems wrong imo... lets hold off on completing this pr then

Reply:
feels wrong to have both, maybe if you pass in backstageItems, it auto adds the BackstageProvider under the hood?

Reply:
In the latest code, the BackstageItemsProvider is no longer used... and from what I can see backstageItems currently only serves to start the BackstageComposer, but it does not provide it the backstage items... I guess we screwed up somewhere in the refactor :)

Below is the only place where we "do" something with backstageItems value, other than passing it on down the chain.
![image](https://user-images.githubusercontent.com/1904889/184444962-4322f654-0a8b-4191-ac31-06d9e369abae.png)

Reply:
I was supposed to remove this prop altogether in the 3.0 release, forgot to scrub TODO comments :/
https://github.com/iTwin/viewer/blob/master/packages/modules/viewer-react/src/components/iModel/IModelViewer.tsx#L20

Reply:
I know we've had previous discussions with the team about not pushing backstages because its really never caught on and has been very confusing. Since the user can provide their own custom frontstage wonder if we should do anything about this. Maybe we should chat about this in next week ui/viewer call?

Reply:
If anything, we should validate what the <BackstageComposer /> does, and if it really need to be removed if there is not backstage items provided. It feels like it should simply always be there and handle UIProvided items, and it would be up to the user to provide its own frontstage that includes the backstage button (where they have access to the full props anyway)